apps/pie-docs/package.json

```diff
@@ -36,6 +36,8 @@
     "markdown-it-attrs": "4.1.6"
   },
   "dependencies": {
+    "@docsearch/css": "3",
```

Comment:
we may want to specify specific 3.x.x versions here - I simply followed the readme which was `3`

Reply:
will also change in next PR 👍 also, checked accessibility with voiceover and all is good!